data/json/mapgen/bridges.json

```diff
@@ -0,0 +1,164 @@
+[
+  {
+    "type": "palette",
+    "id": "bridge_palette",
+    "terrain": {
+      "s": "t_water_moving_sh",
+      "~": "t_water_moving_dp",
+      " ": "t_open_air",
+      ".": "t_pavement_bg_dp",
+      ":": "t_pavement_y_bg_dp",
+      "_": "t_sidewalk_bg_dp",
+      "=": "t_guardrail_bg_dp",
```

Comment:
Not necessary a blocker, but we need to use a different palette (or different terrains) for non-ground level bridges - terrains with `_dp` suffix are being transformed into `t_water_dp` terrain when broken, but they should actually turn into some kind of open air terrain.

Reply:
Also a bit unrelated, but important (as I tried to guerilla the bridge): something is off with explosion resistance of terrains - exploded dynamite, ANFO or even a mininuke (!) charge breaks only a single tile (where bomb was located) at best. 

CC: @kevingranade

Reply:
So I added some new highway terrains that are supposed to transform into t_open_air when smashed, but they're not doing that.  I'm not sure why, but I suspect it's because `map::bash()` is doing something done.

I'll swing back to look at that later.  It's super weird and wrong, but it shouldn't hold up this PR.

Reply:
I agree we can address this later.